### PR TITLE
fix: missing input types

### PR DESCRIPTION
**Pull request by nmerget**
## Proposed changes

- Add missing types for input
- Add step for input types with date/time

## Types of changes

-   [x] Bugfix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Refactoring (fix on existing components or architectural decisions)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] Documentation Update (if none of the other choices apply)

### CODE REVIEW

File: packages/components/src/components/input/model.ts [hunk 1]
```diff
@@ -22,18 +22,24 @@ export type DBInputDefaultProps = {
...
+		| 'hidden'
+		| 'month'
 		| 'number'
+		| 'password'
+		| 'range' // TODO: move this to own component
+		| 'search'
 		| 'tel'
+		| 'text'
+		| 'time'
 		| 'url'
```
Comment (annsch) [inline packages/components/src/components/input/model.ts:39]: do we really want to provide input for all those types?

Comment (mfranzke) [reply packages/components/src/components/input/model.ts:39]: why not ?

File: packages/components/src/components/input/model.ts [hunk 2]
```diff
@@ -22,18 +22,24 @@ export type DBInputDefaultProps = {
...
 	dataListId?: string;
 	type?:
-		| 'text'
-		| 'search'
+		| 'color'
+		| 'date'
+		| 'datetime-local'
+		| 'email'
+		| 'file' // TODO: move this to own component
+		| 'hidden'
```
Comment (mfranzke) [inline packages/components/src/components/input/model.ts:30]: I don't necessarily see a reason this for this one, except consistency. `input[type=hidden]` wouldn't be a UI component, but only a visually hidden HTML element.

What do you think?

**Review by mfranzke — APPROVED** [review]

File: packages/components/src/components/input/model.ts (hunk 1 above)
Comment (nmerget) [reply packages/components/src/components/input/model.ts:39]: We need to handle all those properly in the future.
But for now a migration wouldn't be possible if we don't provide them :(

Comment (mfranzke) [reply packages/components/src/components/input/model.ts:39]: @annsch what was the reason for your question on this topic?

**Pull request merged**